### ISSUE TITLE
tag.permalink and category.permalink lead to incorrect URLs

### ISSUE THREAD

**Issue by piedoom** [getzola/zola#201]
Consider I have the following HTML in my `categories.html` template:

```
<a class='header-link' href="{{category.permalink}}">
    <h4 class="title"> {{ category.name }} </h4>
</a>
````

This will link to `http://domain//category/mycategory/`, which is incorrect both due to the extra slash, and the use of the word `category` instead of `categories`.

The link should be: `http://domain/categories/mycategory/`.   A workaround to this issue is to provide the `slug` instead of `permalink`.  The same issue is prevalent in the `tags`, too.

**Comment by piedoom** [comment]
I don't know the project very well and I haven't been able to get it to compile successfully, but I'd think the offending code [is here](https://github.com/Keats/gutenberg/blob/master/components/taxonomies/src/lib.rs#L47) and should be changed to this:

```
let kind_path = if kind == TaxonomyKind::Tags { "tags" } else { "categories" };
config.make_permalink(&format!("{}/{}", kind_path, slug))
```

**Comment by Keats** [comment]
I believe this is already fixed in the `next` branch

**Comment by Keats** [comment]
Fixed in 0.3.0 that is currently being built by travis/appveyor
